jwst/ami/mask_definition_ami.py

```diff
@@ -0,0 +1,127 @@
+#
+#  Module for defining mask geometry in pupil space
+#  Replace old mask_definitions by reading info in nrm_model into a similar object.
+#
```

Comment:
I don't think it's typically considered good style to have descriptions at the top of a file.  Since there is only one class in here, can relevant information be added to the docstring of that class?

Reply:
It looks like most of the files have commented "Module for..." descriptions at the top, so that may have been the style when it was originally written. I can remove them, but in general I think there are a lot of things in this code that don't follow good style and fixing them all would be a much larger endeavor (that would be good to do at some point, but we don't really have the resources to devote to it right now)